tests/make_test_measurements.py

```diff
@@ -0,0 +1,87 @@
+"""Script to generate fake measurements-data for the telcell project."""
```

Comment:
Misschien aan het begin een beetje een uitleg over wat voor neppe data dit script maakt en wat hier anders is dan bij de andere scripts.